### ISSUE TITLE
RTPEngine Compilation Failure on RHEL 7.9 

### ISSUE THREAD

**Issue by praveenchourasia** [sipwise/rtpengine#1300]
I'm trying to compile RTPEngine on RHEL 7.9 and facing compilation error. I've searched through community documents & going by [rtpengine.spec](https://github.com/sipwise/rtpengine/blob/82c741d88c70e9460ae6335698e37f1184c97868/el/rtpengine.spec) I believe RHEL is a supported platform for RTPEngine. 

### Compilation Error for RTPEngine Release [mr9.4.1.4](https://github.com/sipwise/rtpengine/tree/mr9.4.1.4) or [mr9.4.1.5](https://github.com/sipwise/rtpengine/tree/mr9.4.1.5)

```
cc -g -Wall -Wstrict-prototypes -pthread -fno-strict-aliasing -std=c99 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include   -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include       -pthread     -I/usr/include/libxml2   -I/usr/include/libxml2     -pthread -I/usr/include/json-glib-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include       -DWITH_IPTABLES_OPTION -I. -I../kernel-module/ -I../lib/ -I../include/ -D_GNU_SOURCE -I/usr/include/ffmpeg   -I/usr/include/ffmpeg   -I/usr/include/ffmpeg   -I/usr/include/ffmpeg   -I/usr/include/ffmpeg     -DWITH_TRANSCODING -I/usr/include/mysql -DRE_PLUGIN_DIR="\"/usr/lib/rtpengine\"" -DRTPENGINE_VERSION="\"git-HEAD-e24bbbd\""   -c -o control_ng.strhash.o control_ng.strhash.c
control_ng.strhash.c:45:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
 __csh_hash (str, len)
 ^
control_ng.strhash.c:89:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
 __csh_lookup_raw (str, len)
 ^
In file included from ../lib/loglib.h:9:0,
                 from ./log.h:6,
                 from ../include/obj.h:93,
                 from ../include/control_ng.h:4,
                 from control_ng.strhash.c:161:
../lib/auxlib.h:275:32: error: unknown type name 'uint32_t'
 INLINE double ntp_ts_to_double(uint32_t whole, uint32_t frac) {
                                ^
../lib/auxlib.h:275:48: error: unknown type name 'uint32_t'
 INLINE double ntp_ts_to_double(uint32_t whole, uint32_t frac) {
                                                ^
make[1]: *** [control_ng.strhash.o] Error 1
rm control_ng.strhash.c
make[1]: Leaving directory `/usr/local/src/rtpengine/daemon'
make: *** [all] Error 2
```

The above problem seems to be occurring due to missing header `<stdint.h>` in file `rtpengine/lib/auxlib.h`. And upon adding that the compilation succeeds. 

Could the above mentioned fix be considered as appropriate fix? Wondering as I didn't face this issue while compiling the same on `Debian 10 Buster`.

### Also, in general is RHEL 7.X supported Platform for RTPEngine & can Production workload be ran on the it? 

_NOTE:_ 

- If required, I can share the compilation steps as well. 

OS Details:

```
NAME="Red Hat Enterprise Linux Server"
VERSION="7.9 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.9"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.9 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.9:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.9
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.9"
```

**Comment by rfuchs** [comment]
RHEL is not "officially" supported, as in we don't use, test, or package for it ourselves. Maintaining for RHEL is a community effort. Generally it should work fine, but there can be bumps occasionally. Therefore if you find something wrong with it, or have something that can be improved, please open a pull request.

**Comment by rfuchs** [comment]
The missing header is already fixed via addaa6ac btw

**Comment by praveenchourasia** [comment]
Thanks @rfuchs.
